src/lib/features/project/project-controller.ts

```diff
@@ -210,6 +211,13 @@ export default class ProjectController extends Controller {
         this.use('/', new ProjectStatusController(config, services).router);
         this.use('/', new FeatureLifecycleController(config, services).router);
         this.use('/', new FeatureLinkController(config, services).router);
+
+        if (this.flagResolver.isEnabled('projectContextFields')) {
```

Comment:
this flag is only resolved at startup time but I still think we should be able to not expose it to all customers yet

Reply:
Yeh, I agree.